const v2 = WebAssembly.Exception;
try { new v2(Int8Array); } catch (e) {}
